src/components/log/LogContainer.tsx

```diff
@@ -41,25 +43,25 @@ export class LogContainer extends React.Component<Props, State> {
   }
 
   public componentDidUpdate() {
-    if (this.props.task.logs !== undefined && this.state.logText === "") {
+    if ((this.props.task.logs !== undefined && this.state.logText === "") || (this.props.task.task_id !== this.state.taskId)) {
```

Comment:
I'm not sure if it was this change or if it was like this before, but having a task selected goes into an infinite loop of requesting task logs

Reply:
I'm not sure if that was happening before either, but I'll look in to it.

Reply:
Are you sure it goes in to an infinite loop? I don't see any evidence of that when I test it on my machine.

Reply:
Fixed by adding a componentDidMount method